graph/debruijnnode.cpp

```diff
@@ -374,16 +373,6 @@ QByteArray DeBruijnNode::getGfaSegmentLine(const QString &depthTag) const
     else if (depthTag == "FC")
```

Comment:
Maybe move other code too? This method didn't use private methods of fields as I see. So, all tags' construction will be together.